TACC/Core-CMS-Custom
# Wp 579 request extensions form

## Overview
Convert Request Extension form to react

## Related
- [WP-579](https://tacc-main.atlassian.net/browse/WP-579)
- requires https://github.com/TACC/Core-CMS/pull/117

## Changes
Added ExtensionForm React component

## Testing
1. http://localhost:8000/submissions/extension-request/
2. Verify this matches form: https://dev.apcd.tacc.utexas.edu/submissions/extension-request/
3. Add/Remove Extension Request Buttons work

## UI
<img width="1184" alt="Screen Shot 2024-10-16 at 1 13 34 PM" src="https://github.com/user-attachments/assets/9ed68fba-e23b-432b-9824-e4220dbdc475">



## Notes
 - validating inputs
 - submitting form
 - dynamic dropdowns (they are hardcoded right now)
